src/modules/acbs/acbs-bundle-information.service.create-bundle-information.test.ts

```diff
@@ -357,5 +358,104 @@ describe('AcbsBundleInformationService', () => {
         await expect(createBundleInformationPromise).rejects.toHaveProperty('errorBody', JSON.stringify(errorBody));
       });
     });
+
+    describe('creating a FacilityFeeAmountTransaction', () => {
+      const facilityIdentifier = valueGenerator.facilityId();
+
+      const facilityFeeAmountTransactionMessage: FacilityFeeAmountTransaction = {
+        $type: 'FacilityFeeAmountTransaction',
+        AccountOwnerIdentifier: valueGenerator.string(),
+        EffectiveDate: valueGenerator.dateTimeString(),
+        FacilityIdentifier: facilityIdentifier,
+        FacilityFeeTransactionType: {
+          TypeCode: valueGenerator.nonnegativeInteger(),
+        },
+        IsDraftIndicator: valueGenerator.boolean(),
+        LenderType: {
+          LenderTypeCode: valueGenerator.enumValue(ENUMS.LENDER_TYPE_CODES),
+        },
+        LimitKeyValue: valueGenerator.acbsPartyId(),
+        LimitType: {
+          LimitTypeCode: valueGenerator.string(),
+        },
+        SectionIdentifier: valueGenerator.string(),
+        SegmentIdentifier: valueGenerator.string(),
+        TransactionAmount: valueGenerator.nonnegativeFloat(),
+      };
```

Comment:
Can we not use the generator for this?

Reply:
Thanks Oscar, generator had object for this.